in_toto/rulelib.go

```diff
@@ -0,0 +1,99 @@
+package in_toto
+
+import (
+  "fmt"
+  "strings"
+  )
+
+var errorMsg string = "Wrong rule format, available formats are:\n\t" +
```

Comment:
I don't like these trailing `\t`s to indent *the next line*.

Reply:
Would you prefer the `\t` on the next line, or should I remove it altogether?

Reply:
On the next line.